### PR TITLE
tweaking the bundle's documentation

### PR DESCRIPTION
**Pull request by xabbuh**
* fixing typos
* fixing code blocks
* fixing rST syntax

### CODE REVIEW

File: Resources/doc/commands/generate_controller.rst [hunk 1]
```diff
@@ -25,37 +25,43 @@ The command can be run in a non interactive mode by using the
...
 
     .. code-block:: bash
 
         $ php app/console generate:controller --controller=AcmeBlogBundle:Post
 
-* ``--actions``: The list of actions to generate in the controller class. This
-  has a format like ``%actionname%:%route%:%template`` (where ``:%template%``
-  is optional):
+``--actions``
+
```
Comment (javiereguiluz) [inline Resources/doc/commands/generate_controller.rst]: This description leaves 1 blank line, but others (like the `--controller` description above) don't.

File: Resources/doc/commands/generate_doctrine_entity.rst [hunk 2]
```diff
@@ -7,48 +7,52 @@ Usage
...
 
 .. code-block:: bash
 
-    php app/console generate:doctrine:entity
+    $ php app/console generate:doctrine:entity
 
-The command can be run in a non interactive mode by using the
-``--no-interaction`` option without forgetting all needed options:
+The command can be run in a non-interactive mode by using the ``--no-interaction``
+option without forgetting all needed options:
```
Comment (javiereguiluz) [inline Resources/doc/commands/generate_doctrine_entity.rst]: `forgetting` seems like a typo, right?

File: Resources/doc/commands/generate_doctrine_form.rst [hunk 3]
```diff
@@ -4,20 +4,22 @@ Generating a New Form Type Class Based on a Doctrine Entity
...
 
 Required Arguments
 ------------------
 
-* ``entity``: The entity name given as a shortcut notation containing the
-  bundle name in which the entity is located and the name of the entity. For
-  example: ``AcmeBlogBundle:Post``:
+``entity``
+
+    The entity name given in  shortcut notation containing the bundle name
```
Comment (javiereguiluz) [inline Resources/doc/commands/generate_doctrine_form.rst]: Two white spaces in --> `in  shortcut`

File: Resources/doc/index.rst [hunk 4]
```diff
@@ -1,7 +1,7 @@
 SensioGeneratorBundle
 =====================
 
-The ``SensioGeneratorBundle`` extends the default Symfony2 command line
+The SensioGeneratorBundle extends the default Symfony command line
```
Comment (javiereguiluz) [inline Resources/doc/index.rst]: I would simplify this line:

```
The SensioGeneratorBundle extends the default Symfony command line
 interface by providing new interactive and intuitive commands for generating [...]
```

by:

```
The SensioGeneratorBundle provides commands for generating [...]
```

**Comment by javiereguiluz** [comment]
:+1: @xabbuh as usual, very nice improvements! I've left some minor comments.

**Comment by xabbuh** [comment]
@javiereguiluz thanks, I've applied your suggestions

**Comment by javiereguiluz** [comment]
Thanks @xabbuh.

**Pull request merged**